internal/keycloakb/passwordgeneration.go

```diff
@@ -102,7 +106,7 @@ func GeneratePasswordFromKeycloakPolicy(policy string) (string, error) {
 			}
 		}
 	}
-	rand.Shuffle(len(pwdElems), func(i, j int) { pwdElems[i], pwdElems[j] = pwdElems[j], pwdElems[i] })
+	mrand.Shuffle(len(pwdElems), func(i, j int) { pwdElems[i], pwdElems[j] = pwdElems[j], pwdElems[i] })
```

Comment:
this shuffle is not using the cryptographic random source but not sure if it really matter ?

Reply:
it should not matter